### PR TITLE
review ak

### PR DESCRIPTION
**Pull request by almarklein**
Very nicely written. Just some small suggestions.

### CODE REVIEW

File: papers/luke_campagnola-vispy/vispy.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ VisPy: Harnessing The GPU For Fast, High-Level Visualization
...
+Despite the rapid growth of the scientific Python stack, one conspicuously absent element is a standard package for high-performance visualization. The de-facto standard for plotting is Matplotlib; however, this package is designed for publication graphics and is not optimized for visualizations that require realtime, interactive performance, or that incorporate large data volumes. Several packages have apeared in the Python ecosystem to fill this gap (vtk, chaco, pyqtgraph, visvis, galry, glumpy, etc.), but none has yet emerged as a serious candidate standard package. Consequently, these projects have spent much of their effort re-solving the same problems, and the ecosystem as a whole suffers from the lack of a focused, collaborative effort.
```
Comment (almarklein) [inline papers/luke_campagnola-vispy/vispy.rst:42]: I removed _very_

File: papers/luke_campagnola-vispy/vispy.rst [hunk 2]
```diff
@@ -119,7 +119,7 @@ Internally, visuals upload their data to graphics memory and implement a shader
...
+VisPy implements a collection of coordinate transformation classes that are used to map between a visual's raw data and its output coordinate system (screen, image, svg, etc.). By offloading coordinate transformations to the GPU along with drawing operations, VisPy makes it possible to stream data directly from its source to the GPU without any modification in Python. Most transforms affect the location, orientation, and scaling of visuals and can be chained together to produce more complex adjustments. Transforms may also be nonlinear, as in logarithmic, polar, and mercator projections, and custom transforms can be implemented easily by defining the forward and inverse mapping functions in both Python and GLSL.
```
Comment (almarklein) [inline papers/luke_campagnola-vispy/vispy.rst:122]: (damn git I only changed two words :(  )
Changed last sentence a bit

File: papers/luke_campagnola-vispy/vispy.rst [hunk 3]
```diff
@@ -150,13 +150,13 @@ The following example summarizes the code that produces the logarithmically-scal
...
+Layer 3 implements common features required for interactive visualization, and is the first layer that requires no knowledge of OpenGL. This is the main entry point for most users who build visualization applications. Although the majority of VisPy's graphical features can be accessed by working directly with its Visual classes (layer 2), it can be confusing and tedious to manage the visuals, coordinate transforms, and filters for a complex scene. To automate this process, VisPy implements a scenegraph |---| a standard data structure used in computer graphics that organizes visuals into a hierarchy. Each node in the hierarchy inherits coordinate transformations and filters from its parent. VisPy's scenegraph allows visuals to be easily arranged in a scene and, in automating control of the system of transformations, it is able to handle some common interactive visualization requirements:
```
Comment (almarklein) [inline papers/luke_campagnola-vispy/vispy.rst:153]: First sentence

**Comment by campagnola** [comment]
Thanks for the quick review!
I had to switch to a different branch name (it is now `2015`) but your changes are merged there.

**Pull request closed without merging**